challenge2/styles.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    /* Colors */
+    --primary-color: #FCFCFC;
+    --complimentary-red: #EF4444;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
+    font-family: "Ubuntu", sans-serif;
+}
+
+header p {
+    font-size: 1.8rem;
+}
+
+.main-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 20px;
+    padding-bottom: 25%;
+    background-color: var(--primary-color);
+    flex-direction: column;
+}
+
+.form-container {
+    width: 100%;
+    padding: 26px 8px 20px 20px;
+}
+
+h1 {
+    font-size: 3rem;
+    font-weight: 500;
+    padding-bottom: 3rem;
+}
+
+.title p {
+    font-family: "Manrope", sans-serif;
+    font-size: 1.6rem;
+    font-weight: 400;
+    padding-bottom: 10px;
```

Comment:
We should stick to the same unit for consistency. We are using `rem` for some paddings and `px` for others, always try to be consistent (doesn't matter too much if you do `px` or `rem`, just do always the same)